index.js

```diff
@@ -18,6 +20,7 @@ exports.handler = function(event, context, callback) {
     apiVersion: apiVersion,
     host: endpoint,
     log: logLevel,
+    requestTimeout: timeout,
```

Comment:
It ran successfully. The API docs say the default is 30s, so I'm guessing they ignore undefined values.